tabbycat/api/permissions.py

```diff
@@ -40,24 +42,26 @@ def get_required_permission(self, view):
         Given a model and an HTTP method, return the list of permission
         codes that the user is required to have.
         """
-        return ({
-            'list': getattr(view, 'list_permission', False),
-            'create': getattr(view, 'create_permission', False),
-            'retrieve': getattr(view, 'list_permission', False),
-            'update': getattr(view, 'update_permission', False),
-            'partial_update': getattr(view, 'update_permission', False),
-            'destroy': getattr(view, 'destroy_permission', False),
-            'delete_all': getattr(view, 'destroy_permission', False),
-            'add_blank': getattr(view, 'create_permission', False),
-            'GET': getattr(view, 'list_permission', False),
-            'POST': getattr(view, 'update_permission', False),
-            'PUT': getattr(view, 'update_permission', False),
-            'PATCH': getattr(view, 'update_permission', False),
-            'DELETE': getattr(view, 'destroy_permission', False),
-        }).get(getattr(view, 'action', view.request.method), False)
+        return (
+            {
+                "list": getattr(view, "list_permission", False),
+                "create": getattr(view, "create_permission", False),
+                "retrieve": getattr(view, "list_permission", False),
+                "update": getattr(view, "update_permission", False),
+                "partial_update": getattr(view, "update_permission", False),
+                "destroy": getattr(view, "destroy_permission", False),
+                "delete_all": getattr(view, "destroy_permission", False),
+                "add_blank": getattr(view, "create_permission", False),
+                "GET": getattr(view, "list_permission", False),
+                "POST": getattr(view, "update_permission", False),
+                "PUT": getattr(view, "update_permission", False),
+                "PATCH": getattr(view, "update_permission", False),
+                "DELETE": getattr(view, "destroy_permission", False),
+            }
+        ).get(getattr(view, "action", view.request.method), False)
```

Comment:
Is there a way to configure it to permit two opening brackets on the same line, where the same brackets are adjacent on both sides?